nodecontrol/algodControl.go

```diff
@@ -86,7 +86,7 @@ func (nc NodeController) ServerURL() (url.URL, error) {
 	if err != nil {
 		return url.URL{}, err
 	}
-	return url.URL{Scheme: "http", Host: addr}, nil
+	return url.URL{Scheme: "https", Host: addr}, nil
```

Comment:
I don't think we can't do this.  People normally local algods over http.

Perhaps we can do something like checking if the file explicitly contains a protocol, if so, use it.  That way people who want to can put an entire `https://algoexplorer.io/v2" in `algod.net` Otherwise, it continues to default to http.

I am unsure if there is any other code that reads algod.net that would have to be similarly updated.

Reply:
Makes sense. Will take it out before submitting

Reply:
Trying something else, just allowing specification of the full URI including protocol. Should still work for `localhost:8080` `127.0.0.1:8080` but should also work for `https://apiendpoint.com`